dbn_upper_bound/python/mputility.py

```diff
@@ -245,5 +244,26 @@ def Ht_AFE_ABC(z, t):
     else:
         return H
 
-
 '''End H_t approx functional eqn (Ht_AFE) block'''
+
+'''Miscellanous useful functions'''
+def Nt(t, T):
+    t, T = mp.mpf(t), mp.mpf(T)
+    Tsmall = T/(4 * mp.pi())
+    N0 = Tsmall * mp.log(Tsmall) - Tsmall
+    extra = (t/16.0) * mp.log(Tsmall)
+    Nt = N0 + extra
```

Comment:
@km-git-acc 

In line 250, we have
"Redefining name 'Nt' from outer scope (line 250) (redefined-outer-name)"

This is because, we have a variable Nt and a function Nt with the same name. This needs to be fixed.